_examples/bundler/docker-entrypoint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+require 'open3'
+
+# non-buffered stdout
+STDOUT.sync = true
+
+GEMFILE = File.read("/data/params/Gemfile").strip
+abort "Gemfile param cannot be empty" if GEMFILE.empty?
+
+GEMFILE_LOCK = File.read("/data/params/Gemfile.lock").strip
+abort "Gemfile.lock param cannot be empty" if GEMFILE_LOCK.empty?
+
+cmd = "bundle install --path /data/artifacts --no-prune --deployment --jobs=4 --gemfile=/data/params/Gemfile"
```

Comment:
Since we're in data, the `/data/` part isn't needed.

Reply:
This is needed, otherwise bundler uses the `--gemfile` option path as the root and create a tree like this:

```sh
├── artifacts
├── cache
└── params
    ├── artifacts
    │   └── ruby
    │       └── 2.3.0
    ├── Gemfile
    └── Gemfile.lock
```